### PR TITLE
Add imagePullPolicy to init container

### PR DESCRIPTION
**Pull request by jadams**
Signed-off-by: John Adams <>

### Description
Add imagePullPolicy to fsgroup-volume init container (also remove trailing whitespace in this file)
 
### Issues Resolved
Issue not filed, but breaks startup of Opensearch when no internet connection is available but container is cached
 
### Check List
- [x] Commits are signed per the DCO using --signoff

For any changes to files within Helm chart directories:
- [x] Helm chart version bumped
- [x] Helm chart `CHANGELOG.md` updated to reflect change

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/helm-charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by gentoo9ball** [comment]
Need this merged as well, thank you

**Comment by TheAlgo** [comment]
Hi @gentoo9ball 

Thanks a lot for your contribution. Welcome to OpenSearch community. For this PR can you please create and issue and attach a link to the PR. I can take a look at this PR post that.

**Comment by gentoo9ball** [comment]
> Hi @gentoo9ball
> 
> Thanks a lot for your contribution. Welcome to OpenSearch community. For this PR can you please create and issue and attach a link to the PR. I can take a look at this PR post that.

Done, #420 Thanks!

**Comment by prudhvigodithi** [comment]
Hey @jadams can you please check the tests, I see they are failing.
@TheAlgo can you please review this?
Thank you

**Review by TheAlgo — CHANGES_REQUESTED** [review]
@jadams You may need to fix chart versions to resolve lint errors.

**Comment by jadams** [comment]
@TheAlgo version bumped to match current main

**Review by TheAlgo — APPROVED** [review]
LGTM!. @prudhvigodithi Can you please take a look as well?

File: charts/opensearch/templates/statefulset.yaml [hunk 1]
```diff
@@ -228,12 +228,13 @@ spec:
 {{- if and .Values.persistence.enabled .Values.persistence.enableInitChown }}
       - name: fsgroup-volume
         image: "{{ template "opensearch.dockerRegistry" . }}{{ .Values.persistence.image | default "busybox" }}:{{ .Values.persistence.imageTag | default "latest" }}"
+        imagePullPolicy: "{{ .Values.image.pullPolicy }}"
```
Comment (smlx) [inline charts/opensearch/templates/statefulset.yaml:231]: Should this be `.Values.persistence.imagePullPolicy`?

Comment (gentoo9ball) [reply charts/opensearch/templates/statefulset.yaml:231]: Not according to line 245/246, the opensearch container imagePullPolicy gets set to Values.image.pullPolicy

**Comment by TheAlgo** [comment]
@prudhvigodithi @peterzhuamazon Can you guys please take a look?

**Comment by prudhvigodithi** [comment]
Thanks @jadams LGTM? Can you please backport this PR to 1.x?

**Pull request merged**